static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -168,6 +168,14 @@ export function QuickContextHoverWrapper(props: ContextProps) {
   const [data, setData] = useState<Group | null>(null);
   const delayOpenTimeoutRef = useRef<number | undefined>(undefined);
 
+  useEffect(() => {
+    return () => {
+      // Prevents conflicts in GroupStore usage as we navigate to issue details page
+      // using the shortId link.
+      GroupStore.reset();
+    };
+  }, []);
```

Comment:
I'm still not entirely sure if there's a nicer way to handle this. Could we split this off into a separate PR to follow up on?

Reply:
Done!